smithy-cli/src/test/resources/software/amazon/smithy/cli/commands/upgrade/cases/http-payload.v1.smithy

```diff
@@ -0,0 +1,31 @@
+$version: "1.0"
+
+namespace smithy.example
+
+structure RequiredPayload {
+    @required
+    @streaming
```

Comment:
This should be on the targeted shape and is [only valid on `blob`](https://awslabs.github.io/smithy/1.0/spec/core/stream-traits.html#streaming-trait). Needs updating in a few places and in the upgrade.